### PR TITLE
translation functions document

### PR DESCRIPTION
**Pull request by zhang2014**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Only parts completed, please do not merge. This pr was created to prevent others from doing the same work.

Category:

- Document

### CODE REVIEW

**Comment by blinkov** [comment]
@zhang2014 maybe we should merge at least partial result?

**Comment by zhang2014** [comment]
@blinkov 
There are very few parts that are not completed. maybe it is better to wait for this part to be completed before merging?  And I need to review the documents at least once.

**Comment by blinkov** [comment]
@zhang2014 as you wish, that was just a suggestion

File: docs/en/query_language/functions/ip_address_functions.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
-# Functions for working with IP addresses
+# IP函数
```
Comment (hczhcz) [inline docs/en/query_language/functions/ip_address_functions.md]: A wrong file?
`docs/en/query_language/functions/ip_address_functions.md` -> `docs/zh/query_language/functions/ip_address_functions.md`

**Comment by blinkov** [comment]
@zhang2014 any updates?

**Comment by zhang2014** [comment]
The data types document is not synchronized, the link [here](https://github.com/yandex/ClickHouse/pull/5196/files#diff-b0c444d69c9f36459cd4fe4df5ac271bR183) may report an error?

Maybe we should think about synchronization between documents in different languages.
For example, if documents are found to be updated in CI, create `github issues` by scanning `[来源文章](https://clickhouse.yandex/docs/en/query_language/functions/ip_address_functions/) <!--hide-->` of other documents and finding the contents to be synchronized.

**Comment by zhang2014** [comment]
@blinkov The translation has been completed and can be reviewed @sundy-li

File: docs/zh/query_language/functions/ip_address_functions.md [hunk 2]
```diff
@@ -178,5 +178,68 @@ SELECT IPv6CIDRtoIPv6Range(toIPv6('2001:0db8:0000:85a3:0000:0000:ac1f:8001'), 32
 └────────────────────────────────────────────────────────────────────────────┘
 ```
 
+## toIPv4(string)
```
Comment (zhang2014) [inline docs/zh/query_language/functions/ip_address_functions.md]: The data types document is not synchronized, the link here may report an error?

Maybe we should think about synchronization between documents in different languages.
For example, if documents are found to be updated in CI, create github issues by scanning [来源文章](https://clickhouse.yandex/docs/en/query_language/functions/ip_address_functions/)  of other documents and finding the contents to be synchronized.

File: docs/en/query_language/functions/ip_address_functions.md (hunk 1 above)
Comment (zhang2014) [reply docs/en/query_language/functions/ip_address_functions.md]: Thanks, it has been fixed.

File: docs/zh/query_language/functions/ip_address_functions.md (hunk 2 above)
Comment (blinkov) [reply docs/zh/query_language/functions/ip_address_functions.md]: @zhang2014 we already have some sort of tests for documentation ( https://github.com/yandex/ClickHouse/blob/master/docs/tools/test.py + partially build.py nearby) and there's definitely a lot of room for improvement

Comment (zhang2014) [reply docs/zh/query_language/functions/ip_address_functions.md]: Maybe I can try to implement this? but I'm not sure that's going to happen (I'm not good at this kind of work : ( )

Comment (blinkov) [reply docs/zh/query_language/functions/ip_address_functions.md]: @zhang2014 getting out of comfort zone is always interesting, but maybe there are more exciting ways for you to do so ;)

**Review by sundy-li — APPROVED** [review]

**Comment by sundy-li** [comment]
LGTM, @zhang2014  I can't comment changes due to github bugs.

```
Request URL: https://github.com/yandex/ClickHouse/pull/5196/review_comment/create
Request Method: POST
Status Code: 422 Unprocessable Entity
Remote Address: 192.30.253.113:443
Referrer Policy: no-referrer-when-downgrade
```

Some improvements


格式话 => 格式化
 

code point could be translated to `字符` or `码位`

返回此字符串的Unicode code point长度 =>  返回此字符串的Unicode字符长度      

以code point为单位  =>  以unicode字符为单位

但位置以Unicode code points返回  => 但位置以Unicode为单位计算

**Pull request merged**